Mage.Sets/src/mage/sets/MysteryBooster.java

```diff
@@ -44,3487 +41,1918 @@ public static MysteryBooster getInstance() {
      * - Goblin Trenches added instead of Selesnya Guildmage (See Scryfall note on https://scryfall.com/card/mb1/1695/goblin-trenches )
      * - Prophetic Bolt added instead of Nucklavee (See Scryfall note on https://scryfall.com/card/mb1/1696/prophetic-bolt )
      */
-    protected final Map<Integer, List<CardInfo>> possibleCardsPerBoosterSlot = new HashMap<>();
-
+    protected final Map<Integer, List<CardInfo>> boosterMap = new HashMap<>();
 
     private MysteryBooster() {
         super("Mystery Booster", "MB1", ExpansionSet.buildDate(2019, 11, 7), SetType.SUPPLEMENTAL);
```

Comment:
I would like if we added a new `SetType.REMIX` for that. And support it cleanly in place you need to set it (only tournament settings?), avoiding the need for the 2 `cards.add`, as a REMIX set would be expected to not have its own cards.

Remix is the term wotc used for the https://magic.wizards.com/en/news/mtg-arena/announcing-remix-draft-artifacts-on-mtg-arena that was quite similar: boosters made out of existing cards pulled from various other sets.
Side note if we ever want to implement that one, it could have the similar issues than cube that there could be change each time wotc make that available again in mtgo/arena, so there would be need for versioning ideally (and not make a new Set every time the list change).

Reply:
I can certainly add the new SetType.REMIX, but I don't know how to add the support for it to not have cards. (I think the workaround is okay for now if the solution isn't obvious.)

Reply:
Alright, I'm okay with the current setup as apparently not being able to download all image is worrisome to some users.

I'll try to look into why no card in a set is trouble after that.